### PR TITLE
Module - Articles Most Read - showon

### PR DESCRIPTION
**Pull request by brianteeman**
Summary of Changes

Use the showon functionality now available in forms to simplify/reduce the options displayed in the module options

Testing Instructions

This is only a cosmetic change
Go to the module and try all the options on that page to ensure that the new show/hide functionality makes sense.

### CODE REVIEW

File: modules/mod_articles_popular/mod_articles_popular.xml [hunk 1]
```diff
@@ -72,6 +72,7 @@
 				<field name="date_field" type="list" default="a.created"
 					label="MOD_POPULAR_FIELD_DATEFIELD_LABEL"
 					description="MOD_POPULAR_FIELD_DATEFIELD_DESC"
+					showon="date_filtering:range, relative"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_popular/mod_articles_popular.xml]: please remove the extra space to avoid problems

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on f0ec38f6171fb53dff4b4bccc825868538de4b70<br /><br />Test with success but please read fix the extra space<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10162">issues.joomla.org/joomla-cms/10162</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on c1f6bf9024106dbf5d573ed291f42c18597307a1<br /><br />:wink: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10162">issues.joomla.org/joomla-cms/10162</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on c1f6bf9024106dbf5d573ed291f42c18597307a1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10162">issues.joomla.org/joomla-cms/10162</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10162">issues.joomla.org/joomla-cms/10162</a>.</sub>

**Pull request merged**